edx_reverification_block/xblock/static/js/skip_reverification.js

```diff
@@ -1,25 +1,34 @@
-function SkipReverifcation(runtime, element) {
-  $(element).find('#skip').bind('click', function() {
-      $(element).find('#skip-reverification-modal').show();
+function SkipReverification(runtime, element) {
+
+  var setSkipConfirmVisible = function(isVisible) {
+    var skipConfirm = $(element).find('.reverify-skip-confirm-wrapper'),
+        reverifyMsg = $(element).find('.reverify-now-wrapper');
+    if (isVisible) {
+      reverifyMsg.addClass('hidden');
+      skipConfirm.removeClass('hidden');
+    } else {
+      skipConfirm.addClass('hidden');
+      reverifyMsg.removeClass('hidden');
+    }
+  };
+
+  $(element).find('.reverify-skip-link').on('click', function() {
```

Comment:
$(element) is used a decent amount of times in this function, I would be tempted to assign it as a variable for a minor performance gain.